lib-multisrc/mangathemesia/src/eu/kanade/tachiyomi/multisrc/mangathemesia/MangaThemesia.kt

```diff
@@ -210,23 +261,42 @@ abstract class MangaThemesia(
     }
 
     protected fun String?.removeEmptyPlaceholder(): String? {
-        return if (this.isNullOrBlank() || this == "-" || this == "N/A") null else this
+        return if (this.isNullOrBlank() || this == "-" || this == "N/A" || this == "n/a") null else this
     }
 
-    open fun String?.parseStatus(): Int = when {
-        this == null -> SManga.UNKNOWN
-        listOf("ongoing", "publishing").any { this.contains(it, ignoreCase = true) } -> SManga.ONGOING
-        this.contains("hiatus", ignoreCase = true) -> SManga.ON_HIATUS
-        this.contains("completed", ignoreCase = true) -> SManga.COMPLETED
-        listOf("dropped", "cancelled").any { this.contains(it, ignoreCase = true) } -> SManga.CANCELLED
-        else -> SManga.UNKNOWN
+    open fun String?.parseStatus(): Int {
+        if (this == null) return SManga.UNKNOWN
+
+        return when (this) {
```

Comment:
Lowercase this?

Reply:
idk if some of the lowercase versions of translations are correct

Reply:
Could test in both AS and an Android device (print/log them). Maybe I could write a lint rule to help with this.